cinema_service/settings.py

```diff
@@ -84,10 +85,21 @@
 # Database
 # https://docs.djangoproject.com/en/4.0/ref/settings/#databases
 
+# DATABASES = {
+#     "default": {
+#         "ENGINE": "django.db.backends.sqlite3",
+#         "NAME": BASE_DIR / "db.sqlite3",
+#     }
+# }
+
```

Comment:
Delete the commented code, there is no need for it. Apply this change in other places as well